blocks/columns/columns.js

```diff
@@ -1,32 +1,63 @@
-/* eslint-disable function-call-argument-newline, object-curly-newline, function-paren-newline */
-import { h3, a } from '../../scripts/dom-helpers.js';
+import {
+  div, h3, a, img,
+} from '../../scripts/dom-helpers.js';
 import { createOptimizedPicture } from '../../scripts/aem.js';
 
-function decorateAbout(col) {
+function decorateAbout(col, index) {
   const h2 = col.querySelector('h2');
   const h3s = col.querySelectorAll('h3');
   const newCol = document.createDocumentFragment();
-  newCol.appendChild(h2);
+
+  // Create header row
+  const headerRow = div({ class: 'header-row' });
+  headerRow.appendChild(h2);
+
+  // Add expand icon
+  const iconSrc = index % 2 === 0 ? '/icons/angle-right-white.svg' : '/icons/angle-right-black.svg';
+  const expandIcon = img({
+    class: 'expand-icon',
+    src: iconSrc,
+    alt: 'Expand',
+  });
+  headerRow.appendChild(expandIcon);
+
+  newCol.appendChild(headerRow);
+
+  // Create content wrapper
+  const contentWrapper = div({ class: 'column-content' });
 
   // create cards
   h3s.forEach((h) => {
     const h3Text = h.textContent;
     const { href } = h.querySelector('a');
-    const img = h.nextElementSibling.querySelector('img');
+    const imgEl = h.nextElementSibling.querySelector('img');
     const text = h.nextElementSibling.nextElementSibling;
-    const card = a({ href },
-      createOptimizedPicture(img.src, img.alt || h3Text, true, [{ width: '80' }]),
+    const card = a(
+      { href },
+      createOptimizedPicture(imgEl.src, imgEl.alt || h3Text, true, [{ width: '80' }]),
       h3(h3Text),
       text,
     );
-    newCol.appendChild(card);
+    contentWrapper.appendChild(card);
   });
 
+  newCol.appendChild(contentWrapper);
+
   // clear existing content
   while (col.firstChild) col.removeChild(col.firstChild);
 
   // append new
   col.appendChild(newCol);
+
+  // Add expand/collapse functionality
+  headerRow.addEventListener('click', (e) => {
+    e.preventDefault();
+    if (window.innerWidth < 900) {
+      setTimeout(() => {
+        col.classList.toggle('expanded');
+      }, 10);
+    }
```

Comment:
Why do we need to have this timeout?

Reply:
Ah, this is when I was experimenting with something. We actually don't need this timeout and ill remove it